examples/todos-flow/components/Link.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import React from 'react'
+
+export type Props = {
+  active: boolean,
+  children?: React$Element<any>,
```

Comment:
@cr0cK To clear some things up...

> I guess names with a $ are supposed to be private

I think what you should consider as private are those special "magic" types like `$Shape<T>` ... `React$` is a namespace to prevent collisions with other loaded libdefs, it doesn't mean that this is private.